tests/lib/src/common_utils.h

```diff
@@ -88,8 +89,18 @@ ov::Tensor fillTensorRandom(T &input) {
     ov::Tensor tensor{input.get_element_type(), input.get_shape()};
     std::vector<U> values(ov::shape_size(input.get_shape()));
 
-    for (size_t i = 0; i < values.size(); ++i)
-        values[i] = 1 + static_cast<U> (rand()) / (static_cast<U> (RAND_MAX / (std::numeric_limits<U>::max() - 1)));
+    std::random_device rd;
+    std::mt19937 gen(rd());
+
+    if (std::is_floating_point<U>::value == true) {
```

Comment:
@evgenytalanin-intel the same may be applicable to `fillBlobRandom` (but looks like it's for API1.0)